base_fileurl_field/fields.py

```diff
@@ -61,7 +61,7 @@ def create(self, record_values):
                         storage_key = self._build_storage_key(fname)
                 if not fname:
                     storage_key = False
-                env['ir.attachment'].sudo().with_context(
+                super().with_env(env).sudo().with_context(
```

Comment:
Are you sure about this? You switched the `create` call from `ir.attachment` to the one of the field.

Reply:
Hum indeed,  rollbacked !